cluster/manifests/zmon-worker/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: "zmon-worker"
+  namespace: "kube-system"
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        application: "zmon-worker"
+        version: "v0.1"
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+        scheduler.alpha.kubernetes.io/tolerations: '[{"key":"CriticalAddonsOnly", "operator":"Exists"}]'
+    spec:
+      containers:
+        - name: zmon-worker
+          image: "pierone.stups.zalan.do/stups/zmon-worker:zv195"
+          env:
+            - name: WORKER_TEAM
+              value: "{{ .ConfigItems.account_alias }}"
```

Comment:
Do we still need this? I can't find anything referencing "team" in zmon-worker...